master/flock_server/static/flock-mpi.js

```diff
@@ -508,3 +510,50 @@ mpi.ireduce = async function (sendArr, op, comm) {
         return local;
     }
 }
+
+
+/**
+ * Each process sends the contents of its sendbuffer to the root process. The root process
```

Comment:
I think this should say sendArr since thats what we're calling it in this library (since JS doesn't really use buffers the same way)